.jenkins/Jenkinsfile

```diff
@@ -48,10 +48,10 @@ pipeline {
                     // this folder is mounted from a volume so will have wrong permissions
                     sh 'sudo chown root:root /home/jenkins/.cache'
                     // prepare environment (install python dependencies etc)
-                    sh 'pip install -r requirements/requirements-py-3.7.txt --cache-dir /home/jenkins/.cache/pip'
+                    sh 'pip install -r requirements/requirements-py-3.8.txt --cache-dir /home/jenkins/.cache/pip'
```

Comment:
Why 3.8 and not 3.9?

Reply:
I think it has to be the native python version (which is 3.8 for ubuntu 20.04)

Reply:
> There are a lot of checks failing and it looks like the change of the Python version requires an update of the Jenkins CI job.

Ehm... are you looking at the first or the second commit?

Reply:
Indeed, I looked at the first commit for the jenkins failure, but there are still these "failed check" annotations, do you know what this is about?
![Screen Shot 2021-09-14 at 09 28 28](https://user-images.githubusercontent.com/1441208/133214351-55f451de-64d6-4825-a04f-67ee97c56f0d.png)

Reply:
I don't know - when I click on the icons in the right top corners it brings to jobs that succeeded just fine.

Also, these tests are unrelated to the dockerfile.

I think it's somehow a leftover from the first commit

Reply:
Ok, I assume it is fine then.